asciidoc/volume1/tf1-ch-10-sdpi-p.adoc

```diff
@@ -581,6 +619,89 @@ Future versions *_MAY_* include options such as the following (not in priority o
+[plantuml, target=target=puml-sdpi-p-discovery-proxy-option-sequence-diagram, format=svg, format=svg]
+....
+include::../plantuml/vol1-figure-sdpi-p-discovery-proxy-option-sequence-diagram.puml[]
+....
+
+##David - how much of the information below is in the current TF-2 & TF-2A detail?  TBD whether the requirements should be here at a more generic level or there.  Theory of Operation and non-transaction-specific requirements should go in this section;
+##
+
+* SOMDS_Provider systems shall support use of the SOMDS_Discovery_Proxy Actor (DEV-46)
+* SOMDS_Consumer systems shall support use of the SOMDS_Discovery_Proxy Actor (DEV-47)
+
+#DESCRIBE DEV-46 usage + requirements#
+
+* IF the Discovery Proxy Option is enabled for a SOMDS_Provider Actor, then it shall use the DEV-46 transaction to update the SOMDS Discovery Proxy Actor on its network presence and departure
+* IF the Discovery Proxy Option is not enabled for a SOMDS_Provider, then it shall use the default "ad hoc" network presence transactions (DEV-23 -24 -34)
+* if a SOMDS_Provider Actor is configured or provisioned for the Discovery Proxy Option, but the proxy system is not available, then the provider will default back to "ad hoc" discovery mode
+
+#DESCRIBE DEV-47 usage + requirements#
+
+* IF the Discovery Proxy Option is enabled for a SOMDS_Consumer Actor, then it shall use the DEV-47 transaction to retrieve the current network presence metadata related to those systems that it is looking to discover (see transaction metadata detail);
+* IF the Discovery Proxy Option is not enabled for a SOMDS Consumer Actor, then it shall use the default "ad hoc" network presence transactions for discovery (DEV-23 -24 -34);
+* if a SOMDS Consumer Actor is configured / provisioned for the Discovery Proxy Option, but the proxy system is not available, then the consumer will default back to "ad hoc" discovery mode
+
+#DESCRIBE MIXED AD-HOC & PROXY-BASED Discovery + requirements#
+
+* IF both ad-hoc discovery (using the DEV-23 - 24 - 34 transactions) and proxy-based discovery management (using DEV-46 -47 transactions) are utilized in a SOMDS network deployment ...
+* Provider and Consumer systems shall prioritize use of the SOMDS Discovery Proxy Actor and related transactions;
+* Provider systems shall only use the DEV-46 transaction to update the proxy system, and not use the DEV-23 or DEV-34 transactions;
+* Consumer systems shall first use the DEV-47 transaction to determine the presence / absence of a given consumer or set of consumers;
+* IF a provider is unable to discover the desired consumers using the discovery proxy, then it may use the "ad hoc" DEV-24 to probe the presence of systems using multicast;
+* ... #question# ... what about a given provider that has both proxy and "ad hoc" discovery-based consumer connections?  It will always update the proxy but is there anyway to know that a connection was made via "ad hoc" discovery means and not via the proxy? ...
```

Comment:
It doesn't make a difference. Discovery and "connections" are "uncoupled" aka don't know each other.

Reply:
Question resolved ... and removed.  Thanks